PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/helpers/springcloud/SpringCloudAppPropertyView.java

```diff
@@ -852,4 +883,10 @@ private void updateModel(SpringAppViewModel newModel) throws AzureExecutionExcep
             throw new AzureExecutionException("Cannot render property view due to error", ex);
         }
     }
+
+    private void closeEditor() {
+        DefaultLoader.getUIHelper().closeSpringCloudAppPropertyView(project, appId);
+        PluginUtil.showInfoNotificationProject(project,
```

Comment:
Do we need add a notification for close editor?

Reply:
Need since some deletions are from server side.